gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -516,20 +516,19 @@ def run(argv=None):
   # After pipeline is done, create output tables and load AVRO files into them.
   schema_file = _write_schema_to_temp_file(schema)
   suffixes = []
-  total_base_pairs = []
   try:
     for i in range(num_shards):
       suffixes.append(sharding.get_output_table_suffix(i))
-      total_base_pairs.append(sharding.get_output_table_total_base_pairs(i))
+      total_base_pairs = sharding.get_output_table_total_base_pairs(i)
```

Comment:
Lets move this into the if statement.

Reply:
If you don't mind I'd prefer to keep this line here, since we are reading both of these values from the config file.